### ISSUE TITLE
Cannot activate the 'Stripe' extension because it depends on the '.NET Install Tool for Extension Authors' extension,

### ISSUE THREAD

**Issue by auchenberg** [stripe/vscode-stripe#600]
I'm not a .net developer, yet the stripe extension wants me to install the .net extension.

> Cannot activate the 'Stripe' extension because it depends on the '.NET Install Tool for Extension Authors' extension, which is not installed. Would you like to install the extension and reload the window?

<img width="463" alt="Screen Shot 2022-05-14 at 8 56 43 AM" src="https://user-images.githubusercontent.com/173559/168439474-619934da-e9af-43e6-a822-1edb59d29734.png">

**Comment by gracegoo-stripe** [comment]
Hey Kenneth, we should only be displaying that prompt if we detect any .NET files in your project. Do you happen to have any .sln or .csproj files in your project anywhere? Here are the folders we exclude from this check for reference: https://github.com/stripe/vscode-stripe/blob/master/src/languageServerClient.ts#L341

**Comment by auchenberg** [comment]
@gracegoo-stripe I'm getting the error when trying to launch a GitHub Codespace for this repo https://github.com/auchenberg/kenneth-io, which doesn't contain any .net project files.

<img width="1552" alt="Screen Shot 2022-05-25 at 8 08 01 PM" src="https://user-images.githubusercontent.com/173559/170407827-ff87e0dd-cd83-4f76-bffd-b0fa57d42e04.png">

**Comment by gracegoo-stripe** [comment]
Yeah, I don't see anything that should be triggering the language server. Could you go to Output -> Stripe Language Client? That should have some logs on why the client thought we should launch the C# language server

**Comment by raky291** [comment]
Stripe automatically installs the .NET extension.

I agree; it shouldn't be forced to install a .NET extension to use Stripe.

<img width="1416" height="707" alt="Image" src="https://github.com/user-attachments/assets/9a1f6377-a7b2-4a41-8542-79df73b3891c" />
